Source/WebCore/style/StyleResolveForFont.cpp

```diff
@@ -422,7 +422,10 @@ FontVariationSettings fontVariationSettingsFromCSSValue(const CSSValue& value, c
     }
 
     FontVariationSettings settings;
-    for (Ref item : downcast<CSSValueList>(value)) {
+    auto list = dynamicDowncast<const CSSValueList>(value);
```

Comment:
```suggestion
    auto list = dynamicDowncast<CSSValueList>(value);
```
we don't usually put const in these downcasts